source/GameInterface/Services/Settlements/Patches/DisableSettlementClaimantCampaignBehavior.cs

```diff
@@ -1,12 +0,0 @@
-﻿using HarmonyLib;
-using TaleWorlds.CampaignSystem.CampaignBehaviors;
-
-namespace GameInterface.Services.Settlements.Patches;
-
-
-[HarmonyPatch(typeof(SettlementClaimantCampaignBehavior))]
-internal class DisableSettlementClaimantCampaignBehavior
```

Comment:
Re-enabling this also requires syncing of MapEvent.FinalizeEvent(), Kingdom.RemoveDecision(), Settlement.ItemRoster.AddToCounts(), and Settlement.Stash.Clear(). Create issues for each of these and any that make sense to sync with them, i.e. AddDecision